src/dangerousSetInnerHTMLWarn/index.js

```diff
@@ -20,10 +20,10 @@ const dangerouslySetInnerHTMLDetected = (str) =>
  * Warn if detect "dangerouslySetInnerHTML" added.
  * @param {String[]} files - list of modified files.
  * @param {Function} diffForFile - danger diffForFile.
- * @param {Function} warn - danger warn.
+ * @param {Function} fail - danger fail.
```

Comment:
it's a warn!! not a fail! 😂, warn was good

Reply:
I disagree

Reply:
![](https://user-images.githubusercontent.com/516342/70145436-bf6b0480-16a8-11ea-8135-612f9de0b3a2.png)
A specific route which got a `binding.pry` by mistake. This specific route had no response time alert and was discovered by chance

Reply:
The last drop it **removing binding.pry** only